uspace/lib/c/generic/rtld/module.c

```diff
@@ -180,13 +180,13 @@ module_t *module_load(rtld_t *rtld, const char *name, mlflags_t flags)
 {
 	elf_finfo_t info;
 	char name_buf[NAME_BUF_SIZE];
-	module_t *m;
+	module_t *m = NULL;
```

Comment:
Not really necessary as we unconditionally assign to it a couple of lines below.